app/views/products/_modal.html.haml

```diff
@@ -0,0 +1,8 @@
+.modal-header
```

Comment:
What does this modal stands for? I can't understand it by reading only title, I need to do deep dive in files to understand it. Files names should disclose their content